DSS/astm-uam-psu-dss-0.0.0.yaml

```diff
@@ -2169,9 +2226,59 @@ components:
             All volumes of the resource must be encompassed in these extents. However, these extents do not need to match the precise volumes of the resource; a single bounding extent may be provided instead, for instance.
           items:
             $ref: '#/components/schemas/Volume4D'
-        uss_base_url:
+        ris_base_url:
+          $ref: '#/components/schemas/ResourceUssBaseURL'
+        subscription_id:
+          description: "The ID of an existing subscription that the USS will use to keep the operator informed about updates to relevant airspace information. If this field is not provided when the operational intent is in the Activated, Nonconforming, or Contingent state, then the `new_subscription` field must be provided in order to provide notification capability for the operational intent.  The subscription specified by this ID must cover at least the area over which this operational intent is conducted, and it must provide notifications for operational intents."
+          anyOf:
+          - $ref: '#/components/schemas/EntityID'
+        new_subscription:
+          description: "If an existing subscription is not specified in `subscription_id`, and the operational intent is in the Activated, Nonconforming, or Contingent state, then this field must be populated.  When this field is populated, an implicit subscription will be created and associated with this operational intent, and will generally be deleted automatically upon the deletion of this operational intent."
```

Comment:
The descriptions for subscription in the resource reference mention the operational intent. Was this intentional?